ios/brave-ios/App/Client.xcodeproj/xcshareddata/xcschemes/ActionExtension.xcscheme

```diff
@@ -73,7 +73,6 @@
       savedToolIdentifier = ""
       useCustomWorkingDirectory = "NO"
       debugDocumentVersioning = "YES"
-      askForAppToLaunch = "Yes"
```

Comment:
this seems unrelated?

Reply:
Yeah. Removed this. Xcode keeps automatically modifying this file every time.